### DISCUSSION TITLE
owid-covid-data-last-updated-timestamp.txt

### DISCUSSION THREAD

**Discussion by DirektX** [owid/covid-19-data#2607] [General]
This file is two months old!
[owid-covid-data-last-updated-timestamp.txt](https://github.com/owid/covid-19-data/blob/master/public/data/owid-covid-data-last-updated-timestamp.txt)

**Comment by edomt** [comment]
@lucasrodes We should probably remove it if it's no longer used anywhere.

**Comment by lucasrodes** [comment]
@DirektX The file is no longer updated, see https://github.com/owid/covid-19-data/issues/2485.

We are removing it to avoid further confusion, thanks for reporting!
